src/test/java/org/folio/service/inventory/InventoryManagerTest.java

```diff
@@ -587,68 +587,12 @@ void shouldCheckIfTheHoldingExistsWhenHoldingIdSpecifiedAndIfExistThenReturnHold
     Location location = new Location().withHoldingId(holdingIdExp).withQuantity(1).withQuantityPhysical(1);
 
     doReturn(succeededFuture(holdingExp)).when(restClient).getAsJsonObject(any(RequestEntry.class), eq(requestContext));
-    String holdingIdAct = inventoryManager.getOrCreateHoldingsRecord(instanceId, location, requestContext).result();
+    String holdingIdAct = inventoryHoldingManager.getOrCreateHoldingsRecord(instanceId, location, requestContext).result();
 
     assertThat(holdingIdAct, equalTo(holdingIdExp));
     verify(restClient, times(1)).getAsJsonObject(any(RequestEntry.class), eq(requestContext));
   }
 
-  @Test
```

Comment:
can these tests be un-deleted?

Reply:
They needed refactoring to mock a static API however IDEA showed that they were not covering additional lines so didn't spent time